Documentation/Troubleshooting/HumanReadableUrls/Index.rst

```diff
@@ -12,10 +12,13 @@ parameter. Just enter *http://www.your-site.example.org/index.php?id=33* to get
 the "Overview" page of the Introduction Package.
 
 This is convenient but not good for visitors nor for proper referencing.
-Human-readable URLs are provided by extension "realurl" (others exist).
-Thus the above-mentioned URL becomes *http://www.your-site.example.org/content-examples/overview/*.
 
-This should work out of the box with the Introduction Package. If it does not
-you may want to deactivate human-readable URLs. The simplest way to achieve
-this is to go to the **ADMIN TOOLS > Extensions** module and deactivate
-extension "realurl".
+Starting from version 9.5, Human-readable URLs are created automatically by TYPO3.
```

Comment:
Actually, thinking about it, this is a little out of place in section "Troubleshooting". Because previously, there was a tip about deactivating realurl if there are problems. This is now missing. So it's not really a "Troubleshooting" section at all. I think we should just move this from "Troubleshooting" to maybe the chapter "Next steps" or somewhere else. I'll try to do that in another commit.

Reply:
ok. Let me know if I could be of help

Reply:
in this section maybe it could be added something like "be sure that the .htaccess file is present" or something like that; albeit this is valid only if you use Apache. It could be better to add the configuration of the various webserver (Apache, IIS...) should be checked

Reply:
It is a little difficult to figure out where to put it. 

What I had been meaning to do as one of my many tasks on my todo list is go over "Getting started" and "installation guide" and make sure they are updated for TYPO3 9. Once I get started on that I might have a better idea of what should be there and we might add some more information. I can't really make decisions on what to put where very well right now because I must admit I have never read either one of these guides entirely. Will do that. 

As for this PR: About the section on sites and URL handling in general: I think it would be a good idea to put it in "Other backend modules" chapter. I am working on that now.